lldb/include/lldb/Target/ThreadList.h

```diff
@@ -27,12 +27,13 @@ class ThreadList : public ThreadCollection {
   friend class Process;
 
 public:
-  ThreadList(Process *process);
+  ThreadList(Process &process);
 
   ThreadList(const ThreadList &rhs);
 
   ~ThreadList() override;
 
+  // Precondition: both thread lists must be belong to the same process.
```

Comment:
Should these be `///` so they're picked up by doxygen?

Reply:
Works for me